### PR TITLE
Task closed_at, backfill task.last_assigned_to

### PR DESCRIPTION
**Pull request by longhotsummer**

### CODE REVIEW

File: indigo_api/migrations/0104_task_closed_at.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+def populate_closed_at(apps, schema_editor):
+    Action = apps.get_model("actstream", "action")
+    Task = apps.get_model("indigo_api", "Task")
+    db_alias = schema_editor.connection.alias
+
+    content_type = ContentType.objects.get_for_model(Task)
+
+    # backfill task.closed_at
+    for action in Action.objects.using(db_alias).filter(verb__in=['approved', 'cancelled'], action_object_content_type_id=content_type.pk):
```
Comment (goose-life) [inline indigo_api/migrations/0104_task_closed_at.py]: should 'closed' be added to this list?

Comment (longhotsummer) [reply indigo_api/migrations/0104_task_closed_at.py]: Yes, it should.

Comment (longhotsummer) [inline indigo_api/migrations/0104_task_closed_at.py]:
```suggestion
    for action in Action.objects.using(db_alias).filter(verb__in=['approved', 'cancelled', 'closed'], action_object_content_type_id=content_type.pk):
```

**Comment by goose-life** [comment]
my only question is whether .save() is now needed in the tests?

**Pull request merged**